platform-sdk/swirlds-jasperdb/src/main/java/com/swirlds/merkledb/MerkleDb.java

```diff
@@ -185,7 +185,7 @@ private static Path getDefaultPath() {
         return defaultInstancePath.updateAndGet(p -> {
             if (p == null) {
                 try {
-                    p = LegacyTemporaryFileBuilder.buildTemporaryFile("merkledb");
+                    p = Files.createTempDirectory(null).resolve(MERKLEDB_COMPONENT);
```

Comment:
I _think_ this should actually be using `FileSystemManager.resolveNewTemp(...)` so that temporary files are stored in the correct (not `/tmp`) location for merkldDB.

Reply:
In order to get the correct location is it ok to rely on the `ConfigurationHolder` (I see it's used to get the `MerkleDbConfig`) in order to get properly configured file manager? Something like this perhaps:
```
    final var config = ConfigurationHolder.getInstance().get();
    p = FileSystemManager.create(config).resolveNewTemp(MERKLEDB_COMPONENT);
```

Reply:
Not sure, to be honest.  Perhaps @lpetrovic05 will know the correct usage?

Reply:
ConfigurationHolder is deprecated, but it seems that MerkleDb still uses it. So I would say its ok in this situation to use it for the `FileSystemManager`. ConfigurationHolder can be hopefully removed at some later date